pkgs/development/compilers/gcc/default.nix

```
@@ -321,7 +321,7 @@ pipe ((callFile ./common/builder.nix {}) ({
     inherit langC langCC langObjC langObjCpp langAda langFortran langGo langD version;
     isGNU = true;
     hardeningUnsupportedFlags =
-      optional (!atLeast11) "zerocallusedregs"
+      optional (!(atLeast11 && (targetPlatform.isx86 || targetPlatform.isAarch64))) "zerocallusedregs"
```

According to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=97680#c10 (the Author implemented the flag)
this should be
```suggestion
      optional (!(atLeast11 && (targetPlatform.isx86_64 || targetPlatform.isAarch64 || targetPlatform.isSparc))) "zerocallusedregs"
```

---

Thanks for finding that, I had a look earlier and wasn't able to find a proper source. I originally tried using `isx86_64` but was scared off by the number of rebuilds - with the confirmation it's for x86_64 let's do that.